RecoBTag/FeatureTools/plugins/DeepDoubleCvLTagInfoProducer.cc

```diff
@@ -0,0 +1,349 @@
+
+#include "FWCore/Framework/interface/Frameworkfwd.h"
```

Comment:
this looks identical to ```DeepDoubleBTagInfoProducer```.
So, the module appears to be unnecessary because ```DeepDoubleBTagInfoProducer``` can do the same job